### DISCUSSION TITLE
How to remove apostrophe when copying a query from google excel spreedsheet to Dbeaver?

### DISCUSSION THREAD

**Discussion by mmmmpnda** [dbeaver/dbeaver#20615] [Q&A] [answered]
The engineers told me to manually delete it however it would be a hassle since there's a lot of data and all data has the apostrophe at the beginning and end so had to remove it to make sure I could run the query. (Btw, I'm not an engineering/IT guy so please respond in layman's terms)

This will be a great help to copy and paste multiple queries without needing to remove the apostrophe every time. Thank you so much!

![image](https://github.com/dbeaver/dbeaver/assets/140013295/6ef8fbcc-c453-4cfa-b901-5931b79d15f4)

**Comment by E1izabeth** [comment, marked as answer]
You can turn on this setting and then paste. Quotes will be automatically removed.
![image](https://github.com/dbeaver/dbeaver/assets/28875055/4afc4552-c7f2-47e3-9704-a21a5111f04f)

**Reply by mmmmpnda** [reply]
It is working! Thank you so much for your help <3
